tao_compiler/mlir/disc/transforms/disc_gpu_kernel_to_blob.cc

```diff
@@ -305,9 +305,24 @@ class GpuKernelToBlobPass
 
     xla::HloModuleConfig config;
     xla::DebugOptions options = xla::GetDebugOptionsFromFlags();
-    // Make sure we use full precision division operations.
-    // We need this to pass the ut for `tf.FloorDiv`.
-    (*options.mutable_xla_backend_extra_options())["-nvptx-prec-divf32"] = "2";
+    // The default value is false currently because the UT of tf.FloorDiv UT
+    // fails if prec-div is not 2.
+    bool use_fast_math = false;
+    tensorflow::ReadBoolFromEnvVar("DISC_CUDA_USE_FAST_MATH", false,
+                                   &use_fast_math);
+    if (use_fast_math) {
+      // prec-div is set to 1 instead of 0 for better precision.
+      (*options.mutable_xla_backend_extra_options())["-nvptx-prec-divf32"] =
+          "1";
+      (*options.mutable_xla_backend_extra_options())["-nvptx-prec-sqrtf32"] =
```

Comment:
What about other math fuctions like exp, log, pow e.t.c.?

Reply:
They should be enabled by default. According to the document, the `use_fast_math` flag of `nvcc` only have these three additional settings.